store/pruning/manager_test.go

```diff
@@ -49,12 +47,16 @@ func (s *PruningTestSuite) TestPruning() {
 	s.manager.Start()
 
 	latestVersion := uint64(100)
+	kvCount := 10
 
-	// write 10 batches
+	// write batches
 	for i := uint64(0); i < latestVersion; i++ {
 		version := i + 1
 		cs := store.NewChangeset()
-		cs.Add([]byte("key"), []byte(fmt.Sprintf("value%d", version)))
+		for j := 0; j < kvCount; j++ {
+			cs.Add([]byte(fmt.Sprintf("key-%d", j)), []byte(fmt.Sprintf("value-%d-%d", version, j)))
```

Comment:
Won't this cause a conflict of keys written given than they'll go kvCount times but be repeated latestVersion times?

Reply:
The combination of `key` and `version` is unique in the storage, it is enough to test pruning.